Paymetheus.Tests.Decred/Sha256Tests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Paymetheus.Decred.Wallet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Paymetheus.Tests.Decred
+{
+    public static class Sha256Tests
+    {
+        [Fact]
```

Comment:
I'll do this later.  This is a bit of work for not a whole lot of benefit.  "It works" at this point and there are other more pressing issues.